packages/loader/driver-utils/src/readAndParse.ts

```diff
@@ -13,9 +14,9 @@ import { IDocumentStorageService } from "@fluidframework/driver-definitions";
  * @param id - the id of the blob to read and parse
  * @returns the object that we decoded and JSON.parse
  */
-export async function readAndParse<T>(storage: Pick<IDocumentStorageService, "read">, id: string): Promise<T> {
-    const encoded = await storage.read(id);
-    const decoded = fromBase64ToUtf8(encoded);
+export async function readAndParse<T>(storage: Pick<IDocumentStorageService, "readBlob">, id: string): Promise<T> {
+    const blob = await storage.readBlob(id);
+    const decoded = blobToString(blob);
     return JSON.parse(decoded) as T;
 }
 
```

Comment:
another example on line 30

Reply:
comment seems to be out of place now (at least for me it shows up in between two functions)

Reply:
I left a comment below!